server/app/views/v1/host_nodes/_host_node.json.jbuilder

```diff
@@ -1,4 +1,5 @@
 json.id node.node_id
+json.token node.token
```

Comment:
The `grid` API also exposes the grid token for everybody...  #2013 splits the grid token out into a separate API, which might also be an option here (`/v1/nodes/:grid/:node/token`)?